extensions/wikia/LanguageWikisIndex/LanguageWikisIndexController.class.php

```diff
@@ -17,8 +18,7 @@ public function index() {
 
 		$this->setVal( 'langWikis', WikiFactory::getLanguageWikis() );
 
-		$isClosed = $this->isClosedWiki();
-		if ( $isClosed ) {
+		if ( $wgIncludeClosedWikiHandler ) {
```

Comment:
When `city_public === -1` then `$wgIncludeClosedWikiHandler === false` while `$this->isClosedWiki() === true`. I'm not sure if it matters but I wanted to have a consistent check for a closed wiki.

In the case of "on active wikis the `/language-wikis` path should redirect to the main page" it's better to use `WikiFactory::isPublic` because we want to redirect only on `city_public === 1`.

Reply:
We've agreed to use `WikiFactory::isPublic()` instead of `$wgIncludeClosedWikiHandler`